packages/stream_chat_persistence/lib/src/db/drift_chat_database.dart

```diff
@@ -55,7 +57,7 @@ class DriftChatDatabase extends _$DriftChatDatabase {
 
   // you should bump this number whenever you change or add a table definition.
   @override
-  int get schemaVersion => 22;
+  int get schemaVersion => 23;
```

Comment:
How do we make sure this version is always at least 1 higher than the current stable?